src/wp-includes/class-walker-nav-menu.php

```diff
@@ -39,6 +39,21 @@ class Walker_Nav_Menu extends Walker {
 		'id'     => 'db_id',
 	);
 
+	/**
+	 * The URL to the privacy policy page.
+	 *
+	 * @since 6.2.0
+	 * @var string
+	 */
+	private $privacy_policy_url = null;
```

Comment:
Since it's only supposed to be a `string` and it will be set as such in the constructor, I think this can be left undefined.

```suggestion
	private $privacy_policy_url;
```